### ISSUE TITLE
Demo crashed unexpectedly

### ISSUE THREAD

**Issue by Sukeysun** [AutonoBot-Lab/BestMan#4]
When I run the demo scripts located in ./Example, such as open_drawer.py and open_fridge.py, the scripts unexpectedly crash. The error message is as follows

> 
b3Printf: shelf_top

[Client] Info: success load scene from ../Asset/Scene/Kitchen.json!
[Visualizer] Info: The video can be found in Examples/log/open_fridge.mp4
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: No inertial data for link, using mass=1, localinertiadiagonal = 1,1,1, identity local inertial frame
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: base_footprint
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: No inertial data for link, using mass=1, localinertiadiagonal = 1,1,1, identity local inertial frame
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: cylinder_link
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: No inertial data for link, using mass=1, localinertiadiagonal = 1,1,1, identity local inertial frame
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: plate_link
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: No inertial data for link, using mass=1, localinertiadiagonal = 1,1,1, identity local inertial frame
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: ee_link
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: No inertial data for link, using mass=1, localinertiadiagonal = 1,1,1, identity local inertial frame
b3Printf: b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:

b3Printf: link_tcp
[OMPL Planner] Info: Start planning...

Process finished with exit code 139 (interrupted by signal 11:SIGSEGV)
> 

It seems there may be an issue with this [line](https://github.com/AutonoBot-Lab/BestMan_Pybullet/blob/0.1.0/Motion_Planning/Manipulation/OMPL_Planner/OMPL_Planner.py#L219)

**Comment by starry521** [comment]
First, please check whether the system is Ubuntu. In the previous test, we encountered the problem of conflict between ompl in ROS and ompl in conda. Maybe you can find a computer without ros installed to test, because the previous test has not encountered this error. Looking forward to your further reply.

**Comment by Sukeysun** [comment]
Thank you for your response. The issue was caused by a problem with libompl.so. I ran the following command:
sudo find / -name libompl.so
This revealed that there were multiple versions of libompl.so in my environment, including one within the BestMan virtual environment directory. To address this, I set the LD_PRELOAD environment variable in the terminal to reference the specific libompl.so file located in the BestMan virtual environment's directory:
export LD_PRELOAD="{yourpath}/BestMan/lib/python3.8/site-packages/libompl.so:$LDPRELOAD"
After making this change, the issue was resolved！
